LED_Party/main.py

```diff
@@ -0,0 +1,58 @@
+""" starts an LED party on your badge
+"""
+___name___         = "LED Party (Party Party)"
+___license___      = "MIT"
+___dependencies___ = ["wifi", "http", "ugfx_helper", "sleep"]
+___categories___   = ["LEDs"]
+___bootstrapped___ = True # Whether or not apps get downloaded on first install. Defaults to "False", mostly likely you won't have to use this at all.
```

Comment:
Please set this to false - we can't have too many apps in bootstrap, it increases the chance of bootstrap failing.

Reply:
Oooh well spotted. Corrected